package.json

```diff
@@ -12,9 +12,16 @@
   "bugs": {
     "url": "https://github.com/adazzle/react-data-grid/issues"
   },
-  "main": "lib/index.js",
-  "module": "lib/index.js",
-  "types": "lib/index.d.ts",
+  "exports": {
+    "node": {
+      "import": "./lib/bundle.mjs",
+      "require": "./lib/bundle.cjs"
+    }
+  },
```

Comment:
This works for Node 12 (lts) and Node 14 (current):
https://nodejs.org/dist/latest-v12.x/docs/api/esm.html#esm_nested_conditions
https://nodejs.org/dist/latest-v14.x/docs/api/packages.html#packages_nested_conditions